.fcrepo_wrapper

```diff
@@ -0,0 +1,10 @@
+# Place any default configuration for fcrepo_wrapper here
+port: 8984
+enable_jms: false
+
+download_dir: tmp/fcrepo_wrapper
```

Comment:
I think your own `~/.solr_wrapper` and `~/.fcrepo_wrapper` might override these files?  If not, then, yea, got to remove that here.  (Dragged it over from the travis-cache PR.)